### DISCUSSION TITLE
will you ever use vue.js ?

### DISCUSSION THREAD

**Discussion by hamidovz** [rittaschool/ritta#150] [Q&A]
hi ! i looked your projects and i think you two are great people to work with. i would want to contribute to your projects but the thing is i use vue.js on front-end side and express.js on back-end side. if you are interested i am providing my linkedin profile:  [https://www.linkedin.com/in/zaur-hamidov-2944081a5/](url)

**Comment by hamidovz** [comment]
@raikasdev

**Comment by raikasdev** [comment]
Hey! At least for now the project is pretty much dead, as we have got other projects and schoolwork to do and we are not fully sure where to take this project with the current situation of competitors in Finland (for example the government funded [DigiOne-project](https://digione.fi)).

Thank you for your interest. We will reach out to you if we decide to continue this project seriously at some point.

**Reply by hamidovz** [reply]
@raikasdev thank you for your response ! i am willing to work with you in other projects if you have one. just let me know if you use vue.js on front-end side or express.js on back-end.
